### PR TITLE
Feature#20 scaenned places data

### LINKED ISSUE #20: Scanned places data
# REQUIREMENTS
Scanned places data
Store data with AsyncStorage
Send incident to API
Changed some styles

# Acceptance Criteria
List all required acceptance criteria
- [ ] Scanned places data
- [ ] Store data with AsyncStorage
- [ ] Send incident to API
- [ ] Changed some styles

### PR DESCRIPTION
**Pull request by vmagana3**
# Description
1.-Scanned Places
2.-Save local data with async storage
3.-Change some styles
4.-Send incident to API
  
  ## Screenshots 
captura1
captura2
captura3

### CODE REVIEW

File: COVID19/src/components/Input.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-        </View>
-    );
-}
+  return (
+    <View style={styles.inputContainer}>
+      <Icon
+        name={iconName}
+        color={colors.white}
+        size={25}
+        style={{marginStart: 15, alignSelf: 'center'}}
```
Comment (fernandoamz) [inline COVID19/src/components/Input.js]: The in-line styles are not allowed

File: COVID19/src/screens/Locations.js [hunk 2]
```diff
@@ -49,27 +22,35 @@ const Item = ({place, date, time}) => {
...
 };
 
 const Location = ({navigation}) => {
+  const [dataPlaces, setDataPlaces] = useState();
+  useEffect(() => {
+    AsyncStorage.getItem('ScannedPlaces').then((res) => {
+      if (res === null) {
+        AsyncStorage.setItem('ScannedPlaces', JSON.stringify([]));
+      } else {
+        //console.log(JSON.parse(res));
```
Comment (fernandoamz) [inline COVID19/src/screens/Locations.js]: remove this line

File: COVID19/src/screens/Locations.js [hunk 3]
```diff
@@ -49,27 +22,35 @@ const Item = ({place, date, time}) => {
...
+        setDataPlaces(JSON.parse(res));
+      }
+    });
+  }, []);
+
   return (
     <View style={styles.mainMainConatiner}>
       <View style={styles.mainContainer}>
-        <Text style={styles.mainTitle}>Where have you been</Text>
+        <Text style={styles.mainTitle}>Locations</Text>
```
Comment (fernandoamz) [inline COVID19/src/screens/Locations.js]: Put all the text strings on a constants file

File: COVID19/src/screens/TestType.js [hunk 4]
```diff
@@ -12,14 +12,14 @@ const TestType = ({navigation}) => {
       <Text style={styles.txtTitle}> {constans.testType} </Text>
       <ButtonMain
         captionButton={constans.viralTest}
-        img={require('../utils/images/covidsitos.png')}
+        // img={require('../utils/images/covidsitos.png')}
```
Comment (fernandoamz) [inline COVID19/src/screens/TestType.js]: remove this comment if is not neccessary

File: COVID19/src/screens/TestType.js [hunk 5]
```diff
@@ -12,14 +12,14 @@ const TestType = ({navigation}) => {
...
         onPush={() => {
           navigation.navigate('TestDetails');
         }}
       />
       <ButtonMain
         captionButton={constans.bodyTest}
-        img={require('../utils/images/prueba-de-sangre.png')}
+        // img={require('../utils/images/prueba-de-sangre.png')}
```
Comment (fernandoamz) [inline COVID19/src/screens/TestType.js]: Try to all the images import and add the import here.

File: COVID19/src/screens/registerSymptoms.js [hunk 6]
```diff
@@ -10,68 +10,145 @@ import {
...
+              text: 'Cancel',
+              onPress: () => navigation.navigate('Main'),
+              style: 'cancel',
+            },
+            {text: 'OK', onPress: () => navigation.navigate('Main')},
+          ]);
+        }
+      });
+    });
+  }
```
Comment (fernandoamz) [inline COVID19/src/screens/registerSymptoms.js]: Refactor this code, mak it easy to read

**Review by fernandoamz — CHANGES_REQUESTED** [review]

File: COVID19/src/components/Input.js (hunk 1 above)
Comment (hearagmu) [reply COVID19/src/components/Input.js]: fixed: the style is now in its separate file

File: COVID19/src/screens/Locations.js (hunk 2 above)
Comment (hearagmu) [reply COVID19/src/screens/Locations.js]: fixed

File: COVID19/src/screens/Locations.js (hunk 3 above)
Comment (hearagmu) [reply COVID19/src/screens/Locations.js]: fixed

File: COVID19/src/screens/TestType.js (hunk 4 above)
Comment (hearagmu) [reply COVID19/src/screens/TestType.js]: fixed

File: COVID19/src/screens/TestType.js (hunk 5 above)
Comment (hearagmu) [reply COVID19/src/screens/TestType.js]: fixed

File: COVID19/src/screens/registerSymptoms.js (hunk 6 above)
Comment (hearagmu) [reply COVID19/src/screens/registerSymptoms.js]: It is the distribution suggested by the writing plugin, however the "header" was adjusted to facilitate reading.

**Review by fernandoamz — APPROVED** [review]

**Review by Joaquin-Niembro — APPROVED** [review]

**Review by HRuvalcabaP — APPROVED** [review]
Good Job! :3

**Pull request merged**